lib/thirdpartylibs.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+<libraries>
```

Comment:
this needs to be up a dir level

/thirdpartylibs.xml

Reply:
To know if this is working, look at this admin page:

/admin/thirdpartylibs.php